### DISCUSSION TITLE
Cannot create an Integration

### DISCUSSION THREAD

**Discussion by MMJM** [svix/svix-webhooks#1248] [General]
I have got the Docker image of Svix running locally.

I am able to Login fine and create an application like this:
`svix application create '{\"name\": \"TestApp\"}'`

When I grab the app ID and try to create an Integration like this: 
`svix integration create 'app_2dFSt8fO4lLXSoZ1usITKEpkFhg' '{\"name\": \"TestIntegration\"}'`

All I get is an Error: `404 Not Found` error

I have tried this on the SAAS service and it works fine. 

I can list the applications and it certainly exists 

What is going on to throw this error?

Thanks

**Comment by tasn** [comment]
The OSS version doesn't have integrations at the moment (as it relies on a few other things that are not yet implemented there), though I guess we can consider adding it there too! Mind opening a ticket?

**Comment by MMJM** [comment]
@tasn Thanks for the reply Tom. 

I shall open a ticket for this as suggested. 

Can I ask how is the OSS version meant to work without integrations. I'm doing a Zapier integration, and the docs mention using the Integration Key for the Auth Token as it is a long-lived rotatable credential. 

So what should be used for the OSS version?

What else is different in the OSS version to the SAAS version. It would be handy to know before I go too far down the wrong track.
